ui/components/MesheryPatterns.js

```diff
@@ -634,30 +634,37 @@ function MesheryPatterns({
       },
     );
     catalogVisibilityRef.current = catalogVisibility;
-    const fetchCatalogPatterns = fetchCatalogPattern({
-      selector: {
-        search: '',
-        order: '',
-        page: 0,
-        pagesize: 0,
-      },
-    }).subscribe({
-      next: (result) => {
-        catalogContentRef.current = result?.catalogPatterns;
-        initPatternsSubscription();
-      },
-      error: (err) => console.log('There was an error fetching Catalog Filter: ', err),
-    });
 
-    return () => {
-      fetchCatalogPatterns.unsubscribe();
-      disposeConfSubscriptionRef.current?.dispose();
-    };
+    /*
+     Below is a graphql query that fetches the catalog patterns that is published so
+     when catalogVisibility is true, we fetch the catalog patterns and set it to the patterns state
+     which show the catalog patterns only in the UI at the top of the list always whether we filter for public or private patterns.
+     And our rest api already fetch published so for we not rquired this.
```

Comment:
```suggestion
     Meshery's REST API already fetches catalog items with `published` visibility, hence this function is commented out.
```

Reply:
Thanks, @sudhanshutech. Now: why are we leaving the commented code in the code base?